front/components/assistant_builder/ActionsScreen.tsx

```diff
@@ -122,9 +122,10 @@ const DATA_SOURCES_ACTION_CATEGORIES = [
   "TABLES_QUERY",
 ] as const satisfies Array<AssistantBuilderActionConfiguration["type"]>;
 
-const ADVANCED_ACTION_CATEGORIES = ["DUST_APP_RUN"] as const satisfies Array<
-  AssistantBuilderActionConfiguration["type"]
->;
+const ADVANCED_ACTION_CATEGORIES = [
+  "DUST_APP_RUN",
+  "WEBSEARCH",
+] as const satisfies Array<AssistantBuilderActionConfiguration["type"]>;
```

Comment:
Not sure it's an "advanced action" all the opposite actually. Probably worth cutting a new section?